base/CMakeLists.txt

```diff
@@ -157,7 +157,9 @@ execute_process(
     OUTPUT_STRIP_TRAILING_WHITESPACE
 )
 
-if(DISTRO STREQUAL "centos" OR DISTRO STREQUAL "fedora" OR DISTRO STREQUAL "rhel")
+find_program(XMVN_RESOLVE /bin/xmvn-resolve)
+
+if(MXVN_RESOLVE)
```

Comment:
I think it is mispelled.

Reply:
Well that's annoying, thanks for spotting that!